src/components/Menu.jsx

```diff
@@ -18,13 +19,23 @@ export class Menu extends Component {
     }
   }
 
+    async addToOrder(event) {
+      let id = event.target.parentElement.dataset.id
+      let result = await axios.post('/orders', {id: id});
```

Comment:
The api will take "menu_item" as the parameter name, not "id.
https://github.com/CraftAcademy/slowfood-api-team-2-april-2020/commit/d8a3c830b7af43c08b65fd2801cc6e8620f433eb 
The value should be good.